sros2/package.xml

```diff
@@ -13,6 +13,7 @@
   <depend>rclpy</depend>
   <depend>ros2cli</depend>
 
+  <exec_depend>ament_index_python</exec_depend>
```

Comment:
If we're happy with this being a test-only fix, we should change this to a 'test_depend' before we merge